.all-contributorsrc

```diff
@@ -25,6 +25,17 @@
         "code"
       ]
     },
+{
```

Comment:
You have to modify the contributer src file

Reply:
modify in which way?